### PR TITLE
WINUI-3402-API Reference links

### PR DESCRIPTION
**Pull request by MadhumithanJayapathi**
### Description
Added the reference links for the API used in the AvatarView UG

### CODE REVIEW

File: winui/Avatar-view/customization.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ documentation: ug
 
 # Customization in WinUI AvatarView 
 
-The `AvatarView` control offers a range of customization options, including modifying its border, background, and more.
+The [SfAvatarView](https://www.syncfusion.com/winui-controls/avatar-view) control offers a range of customization options, including modifying its border, background, and more.
```
Comment (Manivannan-E) [inline winui/Avatar-view/customization.md:12]: @MadhumithanJayapathi Use this as class API link.

File: winui/Avatar-view/customization.md [hunk 2]
```diff
@@ -9,15 +9,15 @@ documentation: ug
...
 
 ## Border
 
-Customize the `AvatarView` control border using the `BorderBrush` and `BorderThickness` properties.
+Customize the `AvatarView` control border using the [BorderBrush](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.control.borderbrush?view=windows-app-sdk-1.2) and [BorderThickness](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.control.borderthickness?view=winrt-22621) properties.
```
Comment (Manivannan-E) [inline winui/Avatar-view/customization.md:16]: @MadhumithanJayapathi use link for border brush and thickness like this
https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.control.borderthickness

https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.control.borderbrush

File: winui/Avatar-view/customization.md [hunk 3]
```diff
@@ -100,7 +100,7 @@ avatarView.Background = new SolidColorBrush(Colors.Bisque);
 
 ## Gradient background
 
-Use the `LinearGradientBrush` or `RadialGradientBrush` for the Background property to specify a range of colors to apply the gradient background for the AvatarView as shown in the following code example.
+Use the [LinearGradientBrush](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.media.lineargradientbrush?view=winrt-22621) or [RadialGradientBrush](https://learn.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.media.radialgradientbrush?view=winui-2.8) for the [Background](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.control.background?view=winrt-22621) property to specify a range of colors to apply the gradient background for the AvatarView as shown in the following code example.
```
Comment (Manivannan-E) [inline winui/Avatar-view/customization.md:103]: @MadhumithanJayapathi No need to add link for gradient brush

File: winui/Avatar-view/customization.md [hunk 4]
```diff
@@ -146,7 +146,7 @@ avatarView.Background = new LinearGradientBrush()
 
 ## Font
 
-Customize the font of initials in the `AvatarView` control by using the `FontFamily` and `Foreground` properties.
```
Comment (Manivannan-E) [inline winui/Avatar-view/customization.md:149]: @MadhumithanJayapathi API references are wrong, take properties from here.

https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.control

File: winui/Avatar-view/visual-styles.md [hunk 5]
```diff
@@ -48,17 +48,17 @@ avatarView.CornerRadius = new CornerRadius(0,23,0,23);
...
 * Large
 * Medium
 * Small
 * Extra-small
 
-The following code sample illustrates how to set the visual styles of AvatarView to `Circle`.
+The following code sample illustrates how to set the visual styles of AvatarView to [Circle](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Core.AvatarShape.html#Syncfusion_UI_Xaml_Core_AvatarShape_Circle).
 
-N> The default `AvatarShape` is Circle.
+N> The default AvatarShape is Circle.
```
Comment (Manivannan-E) [inline winui/Avatar-view/visual-styles.md:61]: @MadhumithanJayapathi Add link for notes section.

File: winui/Avatar-view/customization.md (hunk 1 above)
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/customization.md]: @Manivannan-E Used class API links

File: winui/Avatar-view/customization.md (hunk 2 above)
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/customization.md]: @Manivannan-E Used the links given.

File: winui/Avatar-view/customization.md (hunk 3 above)
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/customization.md]: @Manivannan-E Removed the links for gradient brush.

File: winui/Avatar-view/customization.md (hunk 4 above)
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/customization.md:149]: @Manivannan-E Removed the wrong API links and added API's from the link provided

File: winui/Avatar-view/visual-styles.md (hunk 5 above)
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/visual-styles.md]: @Manivannan-E Added links for notes section.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**